packaging/installer/kickstart.sh

```diff
@@ -525,6 +526,27 @@ update() {
   fi
 }
 
+uninstall() {
+  get_system_info
+  detect_existing_install
+
+  if [ $INTERACTIVE = 0 ]; then
+    FLAGS="--yes --force"
+  else
+    FLAGS="--yes"
+  fi
+
+  if [ -f "${INSTALL_PREFIX}/usr/libexec/netdata/netdata-uninstaller.sh" ]; then
+    echo "Found existing netdata-uninstaller. Runing it.."
+    ${INSTALL_PREFIX}/usr/libexec/netdata/netdata-uninstaller.sh $FLAGS
```

Comment:
Given when this function gets invoked, `${INSTALL_PREFIX}` is not guaranteed to be set correctly when this code gets executed. 

We also need to run the uninstaller as root if required (prefixed with `${ROOTCMD}`.

My suggestion for resolving this would be to set a flag in the option parsing code when this option is encountered, and then use that to conditionally invoke it in the same spot we invoke `handle_existing_install`, which will ensure that `${INSTALL_PREFIX}`, `${ROOTCMD}` and `${tmpdir}` are all set correctly and usable here.

Reply:
INSTALL_PREFIX will be set correctly, because we are calling detect_existing_install inside the uninstall function

Reply:
That only works though on Linux systems, and only if the user did not specify an install prefix. `detect_existing_install` only really works properly if called _after_ the full option parsing code has been run and `confirm_install_prefix` has been run (also, `confirm_install_prefix` will need to be updated to allow specifying an install prefix when the `--uninstall` option was passed).